infra/ansible/config/cuda_deps.yaml

```diff
@@ -3,12 +3,14 @@
 cuda_deps:
   # List all libcudnn8 versions with `apt list -a libcudnn8`
   libcudnn:
+    "12.3": libcudnn9-cuda-12=9.0.0.312-1
```

Comment:
Also, when pytorch publishes their cuda 12.3, we should align the cudnn version with them.